src/components/MAPI/organizations/AccessControl/guides/BindRolesToSubjectsGuide.tsx

```diff
@@ -0,0 +1,120 @@
+          links={[
+            {
+              label: 'kubectl gs plugin installation',
+              href: docs.kubectlGSInstallationURL,
+              external: true,
+            },
+            {
+              label: 'kubectl gs login command',
+              href: docs.kubectlGSLoginURL,
+              external: true,
+            },
+            {
+              label: 'kubectl create rolebinding command',
+              href: docs.kubectlCreateRoleBindingURL,
+              external: true,
+            },
+            {
+              label: 'kubectl create clusterrolebinding command',
+              href: docs.kubectlCreateClusterRoleBindingURL,
+              external: true,
+            },
+            {
+              label: 'Management API introduction',
+              href: docs.managementAPIIntroduction,
+              external: true,
+            },
+          ]}
+        />
+      }
+      {...props}
+    >
+      <Box direction='column' gap='small'>
+        <LoginGuideStep />
+        <CLIGuideStep
+          title='2. Bind a Role to a user or group to grant access to resources of this organization'
+          command={`kubectl create rolebinding example \\
+          -n ${namespace} --user example@acme.org --role read-all`}
+        >
+          <Text>
+            <strong>Note:</strong> Replace <code>example</code> with a unique
+            name for the <code>RoleBinding</code>. Replace{' '}
+            <code>example@acme.org</code> with the actual user&apos;s ID and{' '}
+            <code>read-all</code> with the name of the <code>Role</code> you
+            want to bind. Be aware that the role must exist in this namespace
+          </Text>
+          <Text>
+            In order to grant access to a <strong>group</strong> of users, use
+            the <code>--group</code> flag instead of the <code>--user</code>{' '}
+            flag.
+          </Text>
+        </CLIGuideStep>
+        <CLIGuideStep
+          title='3. Bind a ClusterRole to a user or group to grant access to resources of this organization'
+          command={`kubectl create rolebinding example \\
+           -n ${namespace} --user example@acme.org --clusterrole read-all`}
+        >
+          <Text>
+            <strong>Note:</strong> Replace <code>example</code> with a unique
+            name for the <code>RoleBinding</code>. Replace{' '}
+            <code>example@acme.org</code> with the actual user&apos;s ID and{' '}
+            <code>read-all</code> with the name of the <code>ClusterRole</code>{' '}
+            you want to bind.
+          </Text>
+          <Text>
+            In order to grant access to a <strong>group</strong> of users, use
+            the <code>--group</code> flag instead of the <code>--user</code>{' '}
+            flag.
+          </Text>
+        </CLIGuideStep>
+        <CLIGuideStep
+          title='4. Bind a Role or CclusterRole to a ServiceAccount or grant access to resources of this organization'
```

Comment:
Small typo in ClusterRole :sweat_smile:
Also, just a slight difference from the specs: "to grant access" v.s. "or grant access", but it affects the meaning of the title.

Reply:
Ah, great to have a native speaker here :)

Reply:
```suggestion
          title='4. Bind a role or cluster role to a service account or grant access to resources of this organization'
```